src/components/MapBase/__tests__/MapBase.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import MAP_OPTIONS from 'shared/services/configuration/map-options';
+import MapBase from '..';
+
+describe('components/Map', () => {
+  it('should render the map', () => {
+    const { getByTestId, queryByText } = render(<MapBase mapOptions={MAP_OPTIONS} />);
+
+    // Map
+    expect(getByTestId('map-base')).toBeInTheDocument();
+
+    // Tile layer
+    expect(queryByText(/Kaartgegevens CC-BY-4.0 Gemeente Amsterdam/)).toBeInTheDocument();
+  });
+
+  it('should render the zoom control', () => {
+    const { container, rerender } = render(<MapBase mapOptions={MAP_OPTIONS} />);
+
+    expect(container.querySelector('button[title="Inzoomen"]')).not.toBeInTheDocument();
+
+    rerender(<MapBase mapOptions={MAP_OPTIONS} hasZoomControls />);
+    expect(container.querySelector('button[title="Inzoomen"]')).toBeInTheDocument();
+  });
+
+  it('should implement isInteractive', () => {
+    /**
+     *  the isInteractive prop uses the intern state of leaflet (Browser.touch) and therefore cannot be tested
+     */
```

Comment:
This functionality can now be tested

Reply:
It looks that is not possible with jest and mocking to trigger the moveStart event in leafled